hasher-matcher-actioner/hmalib/common/actioner_models.py

```diff
@@ -4,15 +4,15 @@
 import json
 import typing as t
 
-from dataclasses import dataclass, fields
+from dataclasses import dataclass, field, fields
 from hmalib.common.logging import get_logger
 from hmalib.models import BankedSignal, MatchMessage
 from requests import get, post, put, delete, Response
 
 logger = get_logger(__name__)
 
 
-@dataclass
+@dataclass(unsafe_hash=True)
```

Comment:
(ignorable): The alternative we discussed offline is making Label's to be frozen=True, but if there's anywhere that is modifying labels after creation it will start erroring